src/redux/Anime-list/episodeSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const initialState = {
+  episodeList: [],
+  loading: false,
+};
+
+export const fetchEpisode = createAsyncThunk('anime/fetchEpisode', async (id) => {
+  const res = await axios.get(`https://api.jikan.moe/v4/anime/${id}/episodes`);
+  return res.data;
+});
+
+export const episodeSlice = createSlice({
+  name: 'episode',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    builder.addCase(fetchEpisode.fulfilled, (state, action) => {
+      state.episodeList = action.payload;
+    });
+  },
+});
+
+export const { increment, decrement, incrementByAmount } = episodeSlice.actions;
```

Comment:
in your live version, some of the items in the details are not able to fetch data from the API  which in turn does not render anything on the screen as seen below. it occurs across several animes please fix this 😄 
![image](https://user-images.githubusercontent.com/60612329/227042940-7532a3cf-e453-4360-8ec7-d786fe5cad7e.png)